

From: J.M. Eubank <john@thesnappy.net>


---
 gcc/ginclude/float.h |   10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/gcc/ginclude/float.h b/gcc/ginclude/float.h
index 4767d7b9d..c53234854 100644
--- a/gcc/ginclude/float.h
+++ b/gcc/ginclude/float.h
@@ -504,4 +504,14 @@ see the files COPYING3 and COPYING.RUNTIME respectively.  If not, see
 
 #endif /* __STDC_WANT_DEC_FP__ */
 
+#if defined (__MINGW32__) && ! defined (_MINGW_FLOAT_H_)
+/* MinGW.org's runtime libraries provide a supplementary float.h, which
+ * must also be included to complement this one.  Ideally that MinGW.org
+ * header should be included first, and it will include this one, but in
+ * a default configuration it doesn't normally happen this way; when we
+ * didn't see it first, include the MinGW.org header now!
+ */
+# include_next <float.h>
+#endif
+
 #endif /* _FLOAT_H___ */
