Include wincrypt.h for libssp

From: J.M. Eubank <john@thesnappy.net>


---
 libssp/ssp.c |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/libssp/ssp.c b/libssp/ssp.c
index 28f3e9cc6..b2e3634a9 100644
--- a/libssp/ssp.c
+++ b/libssp/ssp.c
@@ -65,6 +65,9 @@ see the files COPYING3 and COPYING.RUNTIME respectively.  If not, see
 #ifdef HAVE_SYSLOG_H
 # include <syslog.h>
 #endif
+#if defined (_WIN32) && !defined (__CYGWIN__)
+#include <wincrypt.h>
+#endif
 
 void *__stack_chk_guard = 0;
 
